main/mainTemplate.json

```diff
@@ -13,6 +13,10 @@
         "description": "SasToken access to Azure Storage resources."
       }
     },
+    "enableIPForwarding": {
+      "type": "bool",
+      "defaultValue": "true"
```

Comment:
default to false.

Reply:
We want to activate IP forwarding by default.